mlflow/store/tracking/file_store.py

```diff
@@ -1319,3 +1326,363 @@ def _read_helper(root, file_name, attempts_remaining=2):
                 return _read_helper(root, file_name, attempts_remaining - 1)
 
         return _read_helper(root, file_name, attempts_remaining=retries)
+
+    def start_trace(
+        self,
+        experiment_id: str,
+        timestamp_ms: int,
+        request_metadata: Dict[str, str],
+        tags: Dict[str, str],
+    ) -> TraceInfo:
+        """
+        Start an initial TraceInfo object in the backend store.
+
+        Args:
+            experiment_id: String id of the experiment for this run.
+            timestamp_ms: Start time of the trace, in milliseconds since the UNIX epoch.
+            request_metadata: Metadata of the trace.
+            tags: Tags of the trace.
+
+        Returns:
+            The created TraceInfo object.
+        """
+        request_id = request_metadata.get(SpanAttributeKey.REQUEST_ID)
+        if request_id is None:
+            raise MlflowException("Request ID is required to start a trace.", INTERNAL_ERROR)
+        _validate_experiment_id(experiment_id)
+        experiment_dir = self._get_experiment_path(
+            experiment_id, view_type=ViewType.ACTIVE_ONLY, assert_exists=True
+        )
+        mkdir(experiment_dir, FileStore.TRACES_FOLDER_NAME)
+        traces_dir = os.path.join(experiment_dir, FileStore.TRACES_FOLDER_NAME)
+        mkdir(traces_dir, request_id)
+        trace_dir = os.path.join(traces_dir, request_id)
+        trace_info = TraceInfo(
+            request_id=request_id,
+            experiment_id=experiment_id,
+            timestamp_ms=timestamp_ms,
+            execution_time_ms=None,
+            status=TraceStatus.IN_PROGRESS,
+            request_metadata=request_metadata,
+            tags=tags,
+        )
+        self._save_trace_info(trace_info, trace_dir)
+        return trace_info
+
+    def _save_trace_info(self, trace_info: TraceInfo, trace_dir, overwrite=False):
```

Comment:
It might be helpful to our 11-months-in-the-future selves to have a visual representation of the directory structure mapping of the input TraceInfo components (similar to how we map out the file structure for a saved model 
|- experiment_id
|---- traces
|--------request_metadata
|--------tags
|--------request_id
.... etc.... (plus we could use that for the docs! ;) )